templates/site/js/node.js

```diff
@@ -134,13 +132,18 @@ document.addEventListener("DOMContentLoaded", function () {
                 const icon_img = document.createElement("img")
                 icon_img.setAttribute("src", link.href)
 
-                const background_elem = document.getElementById("background-icon");
-                background_elem.style.backgroundImage="url(" + link.href + ")";
-                background_elem.classList.add("info-background", "background-node-logo");
+                const image_left = document.getElementById("image-left")
```

Comment:
There is no element with id "image-left"

Reply:
Removed this code